packages/kit/src/core/postbuild/analyse.js

```diff
@@ -128,10 +133,21 @@ async function analyse({ manifest_path, env }) {
 				validate_page_exports(page.universal, page.universal_id);
 			}
 
-			prerender = get_option(nodes, 'prerender') ?? false;
+			prerender = get_option(nodes, 'prerender') ?? prerender ?? false;
 
 			config = get_config(nodes);
-			entries ??= get_option(nodes, 'entries');
+			entries = get_option(nodes, 'entries') ?? entries;
+		}
+
+		if (prerender && route.endpoint && route.page) {
```

Comment:
This doesn't account for the `"auto"` case
```suggestion
		if (prerender === true && route.endpoint && route.page) {
```
(though as stated in the earlier comment this might need a slight tweak either way